qualtrics_api.gemspec

```diff
@@ -20,9 +20,9 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = '>= 2.3.0'
+  spec.required_ruby_version = '>= 3.0.6'
```

Comment:
💭  Does the new code run in Ruby 2.x? 

- If they do I don't feel that we need to change this requirement. 
- If they don't, it's for the best that we bump the gem version as well

Reply:
The new test will not pass in Ruby 2.x. Since we're the only consumers of this gem, it made sense to me to bump the ruby version rather than write a more complex test. I think it's a good idea to bump the gem version as well.

Reply:
LG!